server/game/cards/01_SOR/units/Snowspeeder.ts

```diff
@@ -0,0 +1,26 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+import { Location, RelativePlayer, Trait } from '../../../core/Constants';
+
+export default class Snowspeeder extends NonLeaderUnitCard {
+    protected override getImplementationId() {
+        return {
+            id: '1862616109',
+            internalName: 'snowspeeder',
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.addOnAttackAbility({
+            title: 'Exhaust an enemy Vehicle ground unit',
+            targetResolver: {
+                locationFilter: Location.GroundArena,
+                controller: RelativePlayer.Opponent,
+                cardCondition: (card) => card.isUnit() && card.traits.has(Trait.Vehicle),
```

Comment:
No functional issue here but do you mind using `card.hasSomeTrait(Trait.Vehicle)`. That method is more flexible (can accept an array) so I want to use that one consistently so people know it's there